lesson05/assignment/css/_components.content.scss

```diff
@@ -1,13 +1,82 @@
-html {
-    box-sizing: border-box;
+* {
+    font-family: $font;
 }
 
-*,
-*:after,
-*:before {
-    box-sizing: inherit;
+//Style header
+.item1 {
+    grid-area: header;
+    display: flex;
+    background-color: $red;
+    color: white;
+    justify-content: center;
+    padding: 3.5rem;
 }
 
-.main {
-    padding: 2rem;
+//Style nav
+.item2 {
+    grid-area: nav;
+    display: flex;
+    background-color: $green;
+    color: white;
+    justify-content: space-evenly;
+    padding: 1.5rem;
 }
+
+//Style main content
+.item3 {
+    grid-area: content;
+    display: flex;
+    background-color: $orange;
+    color: white;
+    justify-content: center;
+    flex-direction: column;
+    padding-left: 4rem;
+    padding-right: 4rem;
+    padding-top: 10rem;
+    padding-bottom: 10rem;
```

Comment:
Reminder that there are padding shorthands. For this, it could be `padding: 10rem 4rem;`. From https://developer.mozilla.org/en-US/docs/Web/CSS/padding 
"The padding property may be specified using one, two, three, or four values. Each value is a <length> or a <percentage>. Negative values are invalid.

    When one value is specified, it applies the same padding to all four sides.
    When two values are specified, the first padding applies to the top and bottom, the second to the left and right.
    When three values are specified, the first padding applies to the top, the second to the right and left, the third to the bottom.
    When four values are specified, the paddings apply to the top, right, bottom, and left in that order (clockwise).
"